drivers/user_directory_report/app/views/user_directory_report/warehouse_reports/users/report.haml

```diff
@@ -0,0 +1,27 @@
+- content_for :title, _('User Directory Report')
```

Comment:
This can be `_('User Directory')`, report seems unnecessary.

Reply:
✅